src/Glpi/Controller/LegacyFileLoadController.php

```diff
@@ -55,11 +55,58 @@ public function __invoke(Request $request): Response
             throw new \RuntimeException('Cannot load legacy controller without specifying a file to load.');
         }
 
-        $callback = function () use ($target_file) {
-            require $target_file;
-        };
+        \ob_start();
+        $response = require($target_file);
+        $content = \ob_get_clean();
 
-        return new HeaderlessStreamedResponse($callback->bindTo($this, self::class));
+        if ($response instanceof Response) {
+            // The legacy file contains a return value that corresponds to a valid Symfony response.
+            // This response is returned and any output is discarded.
+
+            if ($content !== '') {
+                \trigger_error(
+                    sprintf('Unexpected output detected in `%s`.', $target_file),
```

Comment:
```suggestion
                    sprintf('Sending output (via "echo", raw HTML or other methods) is deprecated in scripts, you should return an instance of "%s" instead. Occurred in `%s`.', Response::class, $target_file),
```

Reply:
Sending output via `echo` or any other method is not deprecated yet. The issue here is that a script returns a `Response` instance and, at the same time, outputs something.